dir.targets

```diff
@@ -50,16 +50,6 @@
     <PackageTargetFramework Condition="'$(IsTestProject)' == 'true'"></PackageTargetFramework>
   </PropertyGroup>
 
-  <!--
-    add the test-runtime project.json to the list of supplemental test data so it will get included
-    in the test archive; the stress test build system depends on it when building test mixes.
-  -->
-  <ItemGroup Condition="'$(IsTestProject)'=='true'">
```

Comment:
@MattGal or @schaabs can correct me but I think this will end up causing some issues for the stress testing.